.github/workflows/ngtcp2-quictls.yml

```diff
@@ -42,11 +42,14 @@ concurrency:
   group: ngtcp2-openssl-${{ github.event.pull_request.number || github.sha }}
   cancel-in-progress: true
 
+permissions: {}
+
 jobs:
   autotools:
     name: ${{ matrix.build.name }}
     runs-on: 'ubuntu-latest'
     timeout-minutes: 60
+    permissions: read-all
```

Comment:
Is this one really necessary? None of the other workflows needs this, including ngtcp2-gnutls and ngtcp-wolfsshl.

Reply:
Good point, it's probably not. I just left it there because I was afraid of breaking some flow :grimacing: -- and read permissions should not bring any problem, since the repository is already public.
But I'll take it out